docker-compose.yml

```diff
@@ -17,6 +17,7 @@ services:
       context: ./
       dockerfile: Dockerfile.dev
     environment:
+      NODE_ENV: development
       MSSQLSERVER_URL: mssql://sa:Password!!11@mssqlserver:1433/Dev
```

Comment:
@ndarilek Is there some db schema that needs to be applied to the `mssqlserver` container?  Sails fails to establish a connection to `mssqlserver`, I think because the `Dev` database does not exist.  
```
2018-09-28 18:18:11.23 Logon       Error: 18456, Severity: 14, State: 38.
2018-09-28 18:18:11.23 Logon       Login failed for user 'sa'. Reason: Failed to open the explicitly specified database 'Dev'. [CLIENT: 172.18.0.3]
```

I am also unable to run the `sails console` to debug, failing on this import https://github.com/loconomics/loconomics-backend/blob/master/config/http.js#L14.
```
error: A hook (`userconfig`) failed to load!
error: 
error: Attempted to `require('/host/config/http.js')`, but an error occurred:
--
Error: Cannot find module 'api/middleware/i18n'
```